test/verify/check-system-journal

```diff
@@ -468,21 +468,26 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
     def testBinary(self):
         b = self.browser
         m = self.machine
-
+        bin_data = (r"Hel\01" * 250) + "a"
         m.execute(r"printf 'MESSAGE=Hello \01 World\nPRIORITY=3\nFOO=bar\nBIN=a\01b\02c\03\n' | logger --journald")
-        self.login_and_go("/system/logs#/?tag=logger&since=@" + self.test_start_time)
-        b.click("#journal-box .cockpit-logline .cockpit-log-message:contains('[13 bytes of binary data]')")
-        b.wait_text(".pf-c-card__title", "[13 bytes of binary data]")
+        m.execute(fr"printf 'MESSAGE={bin_data}\nPRIORITY=3\nFOO=bar\nBIN={bin_data}\n' | logger --journald")
 
         def wait_field(name, value):
             row_sel = "dt:contains('" + name + "')"
             b.wait_visible(row_sel + "+ dd:contains('" + value + "')")
 
-        wait_field("FOO", "bar")
-        wait_field("BIN", "[6 bytes of binary data]")
+        def repeat(msg, bin_msg):
+            b.click(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{msg}')")
+            b.wait_text(".pf-c-card__title", msg)
+            wait_field("FOO", "bar")
+            wait_field("BIN", bin_msg)
+            b.click('.pf-c-breadcrumb li:first')
+            b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{msg}')")
 
-        b.click('.pf-c-breadcrumb li:first')
-        b.wait_visible("#journal-box .cockpit-logline .cockpit-log-message:contains('[13 bytes of binary data]')")
+        self.login_and_go("/system/logs#/?tag=logger&since=@" + self.test_start_time)
```

Comment:
No need to log in again. Please just do `b.go()`

Reply:
this is the first login